include/nbl/builtin/hlsl/workgroup/broadcast.hlsl

```diff
@@ -21,10 +22,8 @@ namespace workgroup
  * and then all invocations access that index.
  */
 template<typename T, class SharedAccessor, bool edgeBarriers = true>
-T Broadcast(in T val, in uint id)
+T Broadcast(in T val, NBL_REF_ARG(SharedAccessor) accessor, in uint id)
```

Comment:
you still have HLSL keywords line `in` peppered around the code, those should be `NBL_REF_CONST_ARG`